services/grafana/client.go

```diff
@@ -297,54 +297,37 @@ func (c *Client) CreateAnnotation(ctx context.Context, tags []string, text strin
 		return "", errors.Wrap(err, "failed to marhal request")
 	}
 
-	u := url.URL{
-		Scheme: "http",
-		Host:   c.addr,
-		Path:   "/api/annotations",
-	}
-
-	// TODO should be updated to use c.do
-
-	resp, err := c.http.Post(u.String(), "application/json", &buf)
-	if err != nil {
-		return "", errors.Wrap(err, "failed to make request")
-	}
-	defer resp.Body.Close() //nolint:errcheck
+	var headers = make(http.Header)
+	headers.Add("authorization", authorization)
 
 	var response struct {
 		Message string `json:"message"`
 	}
-	if err = json.NewDecoder(resp.Body).Decode(&response); err != nil {
-		return "", errors.Wrap(err, "failed to decode JSON response")
+
+	if err := c.do(ctx, "POST", "/api/annotations", headers, buf.Bytes(), &response); err != nil {
+		logrus.Errorln(err)
+		return response.Message, err
```

Comment:
looks like we can extract this message from `err.(*clientError)`

Reply:
so we don't need https://github.com/percona/pmm-managed/pull/364/files#diff-263e7c0c71ff546b932cc88b2460f68eR135